### PR TITLE
Weekly update of dependencies

### PR DESCRIPTION
**Pull request by danyeaw**

### CODE REVIEW

**Comment by danyeaw** [comment]
@nacho It looks like glib-networking was failing to build because it couldn't find libproxy. I disabled it, since environment proxy should be working in Windows now. Is this OK, or do you think libproxy is needed and I should add a gvsbuild project?

**Comment by nacho** [comment]
> @nacho It looks like glib-networking was failing to build because it couldn't find libproxy. I disabled it, since environment proxy should be working in Windows now. Is this OK, or do you think libproxy is needed and I should add a gvsbuild project?

I think it should be fine, afaik we were never building the libproxy extension

File: gvsbuild/projects/ffmpeg.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def __init__(self):
         Project.__init__(
             self,
             "nv-codec-headers",
-            version="11.1.5.2",
+            version="12.0.16.0",
```
Comment (nacho) [inline gvsbuild/projects/ffmpeg.py:72]: this is tricky, this means that we are bumping the required nvidia driver that we need. We should probably keep the latest 11.x version for now

Comment (danyeaw) [reply gvsbuild/projects/ffmpeg.py]: Thanks @nacho, I dropped that commit :+1:

**Pull request merged**